packages/twenty-server/src/database/typeorm-seeds/core/workspaces.ts

```diff
@@ -34,11 +34,11 @@ export const seedWorkspaces = async (
       logo: 'https://twentyhq.github.io/placeholder-images/workspaces/apple-logo.png',
       activationStatus: WorkspaceActivationStatus.ACTIVE,
     },
-    [SEED_TWENTY_WORKSPACE_ID]: {
+    [SEED_ACME_WORKSPACE_ID]: {
       id: workspaceId,
-      displayName: 'Twenty',
-      domainName: 'twenty.dev',
-      inviteHash: 'twenty.dev-invite-hash',
+      displayName: 'Acme',
+      domainName: 'acme.dev',
+      inviteHash: 'acme.dev-invite-hash',
       logo: 'https://twentyhq.github.io/placeholder-images/workspaces/twenty-logo.png',
```

Comment:
Feel free to upload a logo here https://github.com/twentyhq/placeholder-images :)

Reply:
Done